pt_PT/LC_MESSAGES/theatre.po

```diff
@@ -811,7 +811,7 @@ msgid "VIEW_ON_SUDOMEMO"
 msgstr "Ver no Sudomemo"
 
 msgid "WATCH_HISTORY"
-msgstr "Watch History"
+msgstr "Histórico de Visual."
```

Comment:
I don't think a period is necessary here.

Reply:
Same with the other commented explination.